setup.cfg

```diff
@@ -13,8 +13,8 @@ keywords = ipython, jupyter
 classifiers =
     Development Status :: 5 - Production/Stable
     Framework :: Jupyter
-    # Framework :: Jupyter :: JupyterLab
-    # Framework :: Jupyter :: JupyterLab :: 4
+    Framework :: Jupyter :: JupyterLab
+    Framework :: Jupyter :: JupyterLab :: 3
```

Comment:
:+1: 

Probably these were previously commented while waiting for the classifiers to be available.